src/main.rs

```diff
@@ -59,7 +59,7 @@ fn main() {
                      ip=<ip_addr>,mask=<net_mask>,mac=<mac_addr>\"",
                 )
                 .takes_value(true)
-                .min_values(1),
+                .min_values(0),
```

Comment:
Why this change?

Reply:
@rbradford Thanks for so quickly review! 

set to 0 since I really don't want to specify so many arguments which are actually not necessary in the test for vhost-user-net, like net, fs, pmem etc. and I don't see they are the must while setup cloud-hypervisor guest either. Correct me if there is a reason to include them.

Reply:
If you don't want networking, don't pass "--net". If you pass --net you *must* have something with it. Hence the .min_values(1)